common/app/views/fragments/atoms/snippets/guide.scala.html

```diff
@@ -32,5 +32,14 @@
       @Html(item.body)
     </div>
   }
+  @for(
+    img <- guide.image;
+    asset <- img.allImages.headOption;
+    credit <- asset.credit
+  ) {
```

Comment:
Can this logic be moved to the guide atom class? so it can be testable for instance